luisy/targets.py

```diff
@@ -242,6 +245,75 @@ def read(self):
         return self.spark.table(self.table_uri)
 
 
+class AzureBlobStorageTarget(CloudTarget):
+    file_ending = "pkl"
+
+    def __init__(
+            self,
+            outdir=None,
+            blob_name=None,
+    ):
+        self.outdir = outdir
+        self.blob_name = blob_name
+
+    @property
+    def blob_client(self):
+        return self.fs.get_blob_client(
+            blob=self.path,
+        )
+
+    def make_dir(self, path):
+        pass
+
+    def remove(self):
+        if self.exists():
+            self.blob_client.delete_blob()
+
+    @property
+    def path(self):
+        # TODO: Path here is more an identifier that shows up in `.luisy.hashes`
+        return os.path.join(
+            self.outdir,
+            f"{self.fs.container_name}.{self.blob_name}.{self.file_ending}"
+        )
+
+    def exists(self):
+        """
+        Checks whether the Azure Blob exists.
+
+        """
+
+        return self.fs.exists(self.blob_name)
+
+    def write(self, obj):
+        """
+        Write object to Azure Blob Storage
+        Args:
+            obj (object): object that can be serialized using `pickle` and is to be written to
+                the blob output
+        """
+
+        try:
+            serialized_obj = pickle.dumps(obj)
```

Comment:
as discussed synchronically: We may restrict to the case that `obj` is a pyspark-frame and save to parquet files. Then this target would work the same way as a `DeltaTableTarget`.